docs/definition.rst

```diff
@@ -59,17 +59,22 @@ Here is a sample version 3 EE file. To use Ansible Builder 3.x, you must specify
         #   - registry.redhat.io/ansible-automation-platform-23/ee-minimal-rhel8:latest
         #     (needs an account)
 
+    # Custom package manager path for the RHEL based images
+    # options:
+    #  package_manager_path: /usr/bin/microdnf
+
     additional_build_files:
         - src: files/ansible.cfg
           dest: configs
 
     additional_build_steps:
       prepend_base:
+        - RUN echo This is a preprend base command!
         # Enable Non-default stream before packages provided by it can be installed. (optional)
-        - RUN $PKGMGR module enable postgresql:15 -y
-        - RUN $PKGMGR install -y postgresql
+        # - RUN $PKGMGR module enable postgresql:15 -y
```

Comment:
Why comment these out?

Reply:
Postgresql module stream is available in `ee-minimal-rhel8` or downstream images. Also, this is an optional step there is no actual use of installing postgresql.